mandos-go/json/parse/parseScenario.go

```diff
@@ -267,6 +267,8 @@ func (p *Parser) parseTxStep(txType mj.TransactionType, stepMap *oj.OJsonMap) (*
 			if err != nil {
 				return nil, fmt.Errorf("bad tx step id: %w", err)
 			}
+		case "displayLogs":
```

Comment:
Need to also add to the writer.
Also add an example in `examples.scen.json`. This way we can test that parsing works correctly.

Reply:
I added a test. I don't think printing the bool in writer is relevant